tests/test_data/V3_3_0/settings.yaml

```diff
@@ -74,7 +74,7 @@ bootloaders_formats:
       - net
       - ofnet
   x86_64-efi:
-    binary_name: grubx86.efi
+    binary_name: grubx64.efi
```

Comment:
Same as for the migration. This file should never change! Please add it to the one for the next version and if that doesn't exist please create it!